requirements.txt

```diff
@@ -1,4 +1,4 @@
-pytest==3.0.1
+pytest==3.0.2
```

Comment:
The risk is that we pick up a bugfix release that contains a regression, but I'd say it's relatively low risk. If it does happen though, it can be hard to identify exactly what changed as there'll be no audit trail in GitHub.